// as.proto
syntax = "proto3";
import "item_info.proto";
import "user_info.proto";
package as;
option cc_generic_services = false;

message ASRequest{
  string log_id = 1;
  string user_id = 2;
  user_info.UserInfo user_info = 3;
}

message ASResponse {
   message Error {
        uint32 code = 1; 
        string text = 2;
   }
   Error error = 1;
   repeated item_info.ItemInfo item_infos = 2;
}

service ASService {
    rpc as_call(ASRequest) returns (ASResponse);
};

